api/src/api/selfRegRouter.test.ts

```diff
@@ -63,25 +64,25 @@ describe("selfRegRouter", () => {
       expect(dayjs(putCalledWith.lastUpdatedISO).isSame(dayjs(), "minute")).toBe(true);
       expect(dayjs(putCalledWith.dateCreatedISO).isSame(dayjs(), "minute")).toBe(true);
 
-      expect(response.status).toEqual(200);
+      expect(response.status).toEqual(StatusCodes.OK);
       expect(response.body).toEqual({ authRedirectURL: selfRegResponse.authRedirectURL });
     });
 
-    it("returns an error when auth resume fails", async () => {
+    it("returns an INTERNAL SERVER ERROR when auth resume fails", async () => {
```

Comment:
I like this change to the test name. It makes it much more explicit than just saying "an error". We now know what kind of error. That's great!